backend/sftp/options_test.go

```diff
@@ -625,7 +625,7 @@ func generatePrivateKey(passphrase []byte) ([]byte, error) {
 	}
 
 	if len(passphrase) > 0 {
-		pemBlock, err = x509.EncryptPEMBlock(rand.Reader, pemBlock.Type, pemBlock.Bytes, passphrase, x509.PEMCipherAES256)
+		pemBlock, err = ssh.MarshalPrivateKeyWithPassphrase(privateKey, "", passphrase)
```

Comment:
This changed because x509.EncryptPEMBlock( is deprecated which the linter caught.  It hard matters since it was in a test but for completeness, I updated it.